zetaclient/keys.go

```diff
@@ -164,3 +182,28 @@ func (k *Keys) GetPubKeySet() (common.PubKeySet, error) {
 	pubkeySet.Secp256k1 = pubkey
 	return pubkeySet, nil
 }
+
+// GetHotkeyPassword returns the password to be used
+// returns empty if no password is needed
+func (k *Keys) GetHotkeyPassword() (string, error) {
+	if k.GetKeybase().Backend() == ckeys.BackendFile {
+		return getHotkeyPassword()
+	}
+	return "", nil
+}
+
+// getHotkeyPassword retrieves the HOTKEY_PASSWORD environment variable
+// and returns an error if it's not defined or shorter than 8 characters.
```

Comment:
I strongly encourage us to require a password that is longer than 8 characters, especially since it is something that is going to be almost exclusively programmatically used from an environment variable. A longer passphrase will provide more entropy than an exceptionally complex, but only eight character long password. 

I feel like in this instance we should encourage this better practice, even though ultimately the password will get scraped from the environment variable by an attacker. 

Actual protection for the secret likely would only come from a hardware security module.

Reply:
The 8 characters is the requirement for the Cosmos keyring, this is why I put this specific check.

I agree that HSM would be the optimal security model in practice. This PR is mainly to introduce the option to use a different keyring backend than test, we plan to add the HSM feature on top of it.

Reply:
8 characters is probably fine, and if we're matching a Cosmos keyring requirement then perhaps it shouldn't differ